externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/schema/SchemaHelper.scala

```diff
@@ -140,6 +140,8 @@ object SchemaHelper {
     case dt
         if Array(TIMESTAMP_NTZ, DAY_TIME_INTERVAL, YEAR_MONTH_INTERVAL)
           .contains(dt.getClass.getSimpleName) => Some(dt.defaultSize)
+    case dt @ (_: DecimalType) =>
```

Comment:
why do we need @ here？

Reply:
case dt: DecimalType =>
      Some(dt.precision) ？

Reply:
yeah

Reply:
@waywtdcc can you fix this?